test/OpenJ9_Jsr_292_API/src/com/ibm/j9/jsr292/api/MethodHandleAPI_privateLookupIn.java

```diff
@@ -109,8 +109,8 @@ public static void test_privateLookupIn_TargetClass_Itself_UnnamedModule() throw
 		Lookup lookup = MethodHandles.lookup();
 		Class<?> targetClass = lookup.lookupClass();
 		Lookup mhprivateLookupIn = privateLookupIn(targetClass, lookup);
-		Assert.assertEquals(targetClass, mhprivateLookupIn.lookupClass());
-		Assert.assertEquals(Lookup.PRIVATE, mhprivateLookupIn.lookupModes() & Lookup.PRIVATE);
+		Assert.assertEquals(mhprivateLookupIn.lookupClass(), targetClass);
+		Assert.assertEquals(mhprivateLookupIn.lookupModes() & Lookup.PRIVATE, Lookup.PRIVATE);
```

Comment:
Any particular reason to switch the order here? (assuming this is not related to `MH_defineClass` changes.)

Reply:
Assert.assertEquals() in testNG is different from Junit in terms of signature as follows:
Assert.assertEquals(acutal_result, expected_result)

(please take reference to https://jitpack.io/com/github/cbeust/testng/master-6.12-g62083d8-127/javadoc/org/testng/Assert.html if interested)